src/API/Middlewares/HttpExceptionsMiddleware.test.ts

```diff
@@ -0,0 +1,99 @@
+import { IllegalArgumentException, NotFoundException } from '@po.et/poet-js'
+import { describe } from 'riteway'
+import { spy } from 'sinon'
+import { getCodeError, HttpExceptionsMiddleware, executeLogger } from './HttpExceptionsMiddleware'
+
+describe('HttpExceptionsMiddleware', async (should: any) => {
+  describe('HttpExceptionsMiddleware middleware', async (should: any) => {
```

Comment:
remove nested describes, the format does not work with riteway. Check the test output.